sdk/va-report-components/README.md

```diff
@@ -13,22 +13,32 @@ Access to a deployment of SAS Visual Analytics 8.4 (or later) is necessary in or
 
 ### NPM
 
-The <a target="_blank" href="https://www.npmjs.com/package/@sassoftware/va-report-components">`@sassoftware/va-report-components`</a> library is published to NPM and can be installed by running the `npm install` command as shown below. `va-report-components` does not support ES6 imports. Therefore, the contents of the `va-report-components/dist` folder must be deployed with your page, and then loaded using a `script` tag.
+The <a target="_blank" href="https://www.npmjs.com/package/@sassoftware/va-report-components">`@sassoftware/va-report-components`</a> library is published to NPM and can be installed by running the `npm install` command as shown below. `va-report-components` can then either be loaded with a `script` tag or with ES module imports.
```

Comment:
Tweaks for the last sentence: 'va-reports-components' can then be loaded with either a 'script' tag or with ES module imports.